tests/auto/tst_dbmanager/tst_dbmanager.pro

```diff
@@ -1,5 +1,6 @@
 TARGET = tst_dbmanager
```

Comment:
Ah, the assumption that it's enough to remove Makefiles was wrong... Fixed.

For some reason tst_dbmanager fails to build without the mock, but tst_webview on the opposite doesn't need the mock as it uses the real class.